include/experimental/__p0009_bits/macros.hpp

```diff
@@ -101,6 +103,52 @@
 #define MDSPAN_IMPL_STANDARD_NAMESPACE_STRING MDSPAN_PP_STRINGIFY(MDSPAN_IMPL_STANDARD_NAMESPACE)
 #define MDSPAN_IMPL_PROPOSED_NAMESPACE_STRING MDSPAN_PP_STRINGIFY(MDSPAN_IMPL_STANDARD_NAMESPACE) "::" MDSPAN_PP_STRINGIFY(MDSPAN_IMPL_PROPOSED_NAMESPACE)
 
+namespace MDSPAN_IMPL_STANDARD_NAMESPACE {
+namespace detail {
+
+inline void default_precondition_violation_handler(const char* cond, const char* file, unsigned line)
```

Comment:
Fine by me but just noting that the default handler would not work with CUDA/HIP

Reply:
`printf` works just fine with CUDA; it's HIP that's broken lol ; - )

Reply:
it is a `fprintf` and it is qualified so there is no way it works on CUDA.  As far as I know HIP behaves just the same as CUDA here.